tools/deb/description-discovery

```diff
@@ -0,0 +1 @@
+Istio Pilot server provides services discovery for the http://istio.io project.
```

Comment:
copyright missing..
Also, Istio Pilot server provides service discovery and traffic management for the https://istio.io project

Reply:
Updated text, same on copyright.